hipchat/views.py

```diff
@@ -29,11 +29,14 @@ def getEmailAddress(self, hipchat_user_id ):
     def getUser(self, email):
```

Comment:
These methods (`getUser`, `getEmailAddress`, `getEntryFormPost`) should not be in this class. Rather refactor them into a helper. 

Also: these methods names are not pythonic. The python standard for naming methods would be: `get_user`, `get_email_address`, `get_entry_form_post`